amazon-qldb-driver/examples/unique.rs

```diff
@@ -60,11 +62,11 @@ async fn main() -> Result<()> {
     // ```sh
     // qldb> delete from example
     // ```
+    let aws_config = aws_config::load_from_env().await;
+
     let driver = QldbDriverBuilder::new()
         .ledger_name("unique-example")
-        .via_rusoto()
-        .region(Region::UsWest2)
```

Comment:
Is this default moved into the config?

Reply:
The regional default made no sense. The other defaults are in the config.